app/src/main/res/values-zh-rTW/strings.xml

```diff
@@ -94,32 +94,32 @@
 	<string name="no_permission_to_write_text">無法取得寫入權限</string>
 	<string name="no_timestamp_set_message">沒有為歌詞物件設定時間標記組</string>
 	<string name="ok">確定</string>
-	<string name="overwrite_prompt">資料夾 %2$s 中具有相同名稱的檔案 \'%1$s.lrc\' 已經存在。您想要取代此檔案嗎？</string>
+	<string name="overwrite_prompt">資料夾 %2$s 中具有相同名稱的檔案「%1$s.lrc」已經存在。您想要取代此檔案嗎？</string>
 	<string name="paste_lyrics_prompt">在下面貼上歌詞</string>
 	<string name="permission_check_message">請確定您已授予權限</string>
 	<string name="play_button_description">播放/暫停</string>
 	<string name="play_lyric_description">播放此歌詞</string>
 	<string name="playback_options_unsupported_message">播放選項只於 Android Marshmallow (6.0) 版本及以上版本支援</string>
-	<string name="playback_pitch_displayer">音調: %1$d%%</string>
-	<string name="playback_speed_displayer">速度: %1$d%%</string>
+	<string name="playback_pitch_displayer">音調：%1$d%%</string>
+	<string name="playback_speed_displayer">速度：%1$d%%</string>
 	<string name="player_not_ready_message">播放器尚未準備好</string>
 	<string name="player_playback_options_title">媒體播放器播放選項</string>
 	<string name="preparing_the_player_message">正在準備播放器...</string>
 	<string name="previous_description">向後搜索 5 秒</string>
 	<string name="processing">處理中</string>
-	<string name="rate_and_feedback_text">這個App由我自已開發及維護的\n請為這個 App 評分及留下有用的評論\n\n\n非常感謝！\n♥</string>
+	<string name="rate_and_feedback_text">這個 App 由我自已開發及維護的\n請為這個 App 評分及留下有用的評論\n\n\n非常感謝！\n♥</string>
 	<string name="read_location_description">讀取歌詞檔的位置</string>
 	<string name="read_location_extra_description">子目錄亦會被掃描</string>
 	<string name="read_location_invalid_message">讀取位置不存在。已經嘗試於 %1$s 建立資料夾但不成功。您想要將讀取位置重置為預設位置嗎？</string>
-	<string name="read_location_non_existent_message">讀取位置不存在。此外無法建立 "Lyrics" 資料夾。正在嘗試更改讀取位置</string>
+	<string name="read_location_non_existent_message">讀取位置不存在。此外無法建立「Lyrics」資料夾。正在嘗試更改讀取位置</string>
 	<string name="rename">重新命名</string>
 	<string name="rename_failed_message">無法重新命名！</string>
 	<string name="rename_successful_message">成功重新命名</string>
-	<string name="renaming_message">正在重新命名... 這可能需要一些時間，取決於檔案與讀取位置的距離</string>
+	<string name="renaming_message">正在重新命名... \n這可能需要一些時間，取決於檔案與讀取位置的距離</string>
```

Comment:
I don't think the newline character is required here too as the dots provide enough seperation